static/js/components/tiles/line_tile.tsx

```diff
@@ -144,7 +145,36 @@ export function LineTile(props: LineTilePropType): JSX.Element {
       replacementStrings={getReplacementStrings(props)}
       className={`${props.className} line-chart`}
       allowEmbed={true}
-      getDataCsv={chartData ? () => dataGroupsToCsv(chartData.dataGroup) : null}
+      getDataCsv={() => {
+        const denoms = props.statVarSpec.map((v) => (v.denom ? v.statVar : ""));
+        const entityProps = props.placeNameProp
+          ? [props.placeNameProp, ISO_CODE_ATTRIBUTE]
+          : undefined;
+        if ("enclosedPlaceType" in props && props.enclosedPlaceType) {
```

Comment:
nit: I think you only need `props.enclosedPlaceType` here because if the field doesn't exist, `props.enclosedPlaceType` would evaluate to `undefined` which would be falsy?